content/departments/marketing/blog.md

```diff
@@ -74,7 +79,26 @@ The `description` field is used as an excerpt for your post on the blog the inde
 - The `socialImage` field is optional. Use the full path to image in order to be read properly on Twitter and Facebook. Ideal image size: 1,200 x 628 pixels. <a href="https://sproutsocial.com/insights/social-media-image-sizes-guide/" rel="nofollow" target="_blank">Latest social size guidelines</a>.
 - The `videoID` field is an optional YouTube video ID and will take priority even if the `socialImage` is present. This will generate an inline video preview card when sharing on social media. This is supported for all types of posts; blog, podcast, and release posts.
 
-## Adding images and other media
+## Using the Blog Post Starter Pack
+
+The Content Platform Team maintains a template Markdown file that exposes all of the content components available for use in the blog. You can directly copy and paste components from this file into your post.
+
+Access the template in the About repository: `/content/blogposts/blog-post-starter-pack.md`
```

Comment:
I know this file isn't posted yet in the About repo, but it'd be helpful to have a link to the file to be really explicit here

Reply:
Agreed! I can add it once we merge the About PR.